Userland/Applications/ClockSettings/TimeZoneSettingsWidget.h

```diff
@@ -15,6 +15,8 @@
 
 class TimeZoneSettingsWidget final : public GUI::SettingsWindow::Tab {
     C_OBJECT(TimeZoneSettingsWidget)
```

Comment:
I think you also want to replace this with `C_OBJECT_ABSTRACT`, otherwise you'll get some other factory functions that don't do all the needed initialization.

Reply:
Okay, will add that to the commit